### DISCUSSION TITLE
Docs: fromRequest with unknown syntax?

### DISCUSSION THREAD

**Discussion by ahoiroman** [spatie/laravel-data#12] [Q&A]
Hi there,

in the docs we find this example:

```php
    public static function fromRequest(Request $request): static
    {
        return new self(
            {$request->input('title_of_song')}, 
            {$request->input('artist_name')}
        );
    }
```

Neither me nor PHPStorm know what this syntax is supposed to do - and the interpreter throws an error because of the unparseable code, too.

I'd love to fix it, but I'm not even sure what this `{}` is _supposed_ to do at all... Any hints here?

**Comment by rubenvanassche** [comment]
Hi @ahoiroman,

Yeah don't know how that passed my review, this is not a valid syntax and I've updated it!
